ros_ws/src/simulation/racer_description/urdf/racer_plugins.gazebo

```diff
@@ -112,13 +112,13 @@
     <gravity>true</gravity>
     <sensor name="imu_sensor" type="imu">
       <always_on>true</always_on>
-      <update_rate>100</update_rate>
+      <update_rate>10000</update_rate>
       <visualize>true</visualize>
       <topic>__default_topic__</topic>
       <plugin filename="libgazebo_ros_imu_sensor.so" name="imu_plugin">
         <topicName>imu</topicName>
         <bodyName>imu_link</bodyName>
-        <updateRateHZ>10.0</updateRateHZ>
+        <updateRateHZ>1000.0</updateRateHZ>
```

Comment:
Is there any use in having such a high update rate? The phyics and the laser scan are only updated at 60Hz.

Reply:
Because 10Hz turned out to be way too slow I've set it to 1000Hz for testing purposes and kinda forgot to change it back to a reasonable rate :/ I've set it now to 100Hz until we get the real IMU working and see at which frequency it publishes its data.